src/app/app.component.ts

```diff
@@ -10,26 +10,29 @@ export class AppComponent implements OnInit {
   public adjustFooter = false;
   public isMobile: boolean;
   public isHome: boolean;
+  public isPageNotFound: boolean;
 
   constructor(private router: Router) {
     router.events.subscribe((val) => {
       if (val instanceof NavigationEnd) {
         this.isHome = val.url === "/" ? true : false;
+        this.isPageNotFound = val.urlAfterRedirects === "/404" ? true : false;
```

Comment:
I would suggest this if it's still as clear? (it clashes with the line above, but in my view that would be an argument for changing both 😁)
```suggestion
        this.isPageNotFound = val.urlAfterRedirects === "/404";
```

(and similar in `ngOnInit`)